src/main/java/henry/ui/MainWindow.java

```diff
@@ -0,0 +1,52 @@
+package henry.ui;
+
+import henry.Henry;
+import javafx.fxml.FXML;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.image.Image;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+/**
+ * Controller for MainWindow. Provides the layout for the other controls.
+ */
+public class MainWindow extends AnchorPane {
+    @FXML
+    private ScrollPane scrollPane;
+    @FXML
+    private VBox dialogContainer;
+    @FXML
+    private TextField userInput;
+    @FXML
+    private Button sendButton;
+
+    private Henry henry;
+
+    private Image userImage = new Image(this.getClass().getResourceAsStream("/images/DaUser.png"));
```

Comment:
Use of pictures for better GUI is good here. It is better than having just text UI.